app/views/works/new_import.html.erb

```diff
@@ -3,7 +3,7 @@
 <%= error_messages_for :work %>
 <div class="notice">
   <p class="important">
-    <%= ts("Please note! Fanfiction.net does not allow imports from their site.") %>
+    <%= ts("Please note! Fanfiction.net and Quotev.com do does not allow imports from their sites.") %>
```

Comment:
I think you forgot to take out the "does" here -- it currently says "do does not."